### PR TITLE
[deckhouse-controller] In Deckhouse Release, display the status that the Deckhouse update is in progress

### PR DESCRIPTION
**Pull request by libmonsoon-dev**
## Description
Fixed release.deckhouse.io/isUpdating annotation

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse-controller
type: fix
summary: Fixed release.deckhouse.io/isUpdating annotation
impact_level: default
```

### CODE REVIEW

**Comment by libmonsoon-dev** [comment]
image

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by libmonsoon-dev** [comment]
/e2e/abort yandex-cloud;WithoutNAT;containerd;1.27 10141513379 10141513379-1-con-1-27 /sys/deckhouse-oss/install:pr9081 redos@89.169.142.6

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @libmonsoon-dev. I've recognized your '/e2e/abort' command and started the workflow...

File: deckhouse-controller/pkg/controller/deckhouse-release/updater/updater.go [hunk 1]
```diff
@@ -57,17 +60,17 @@ func (w *webhookDataGetter) GetMessage(release *v1alpha1.DeckhouseRelease, relea
...
 }
 
-func newKubeAPI(client client.Client, dc dependency.Container, imagesRegistry string) *kubeAPI {
-	return &kubeAPI{client: client, dc: dc, imagesRegistry: imagesRegistry}
+func NewKubeAPI(client client.Client, dc dependency.Container, imagesRegistry string) *KubeAPI {
+	return &KubeAPI{client: client, dc: dc, imagesRegistry: imagesRegistry}
 }
 
-type kubeAPI struct {
+type KubeAPI struct {
```
Comment (juev) [inline deckhouse-controller/pkg/controller/deckhouse-release/updater/updater.go:67]: Sorry, I'm a bit lost, but for what purpose are we making the structure and the method of creating the structure public?

**Review by miklezzzz — APPROVED** [review]

File: deckhouse-controller/pkg/controller/deckhouse-release/updater/updater.go (hunk 1 above)
Comment (miklezzzz) [reply deckhouse-controller/pkg/controller/deckhouse-release/updater/updater.go:67]: it's reused [here](https://github.com/deckhouse/deckhouse/pull/9081/files#diff-f5f1549dccbd7351a2389d6bc059b5f448e7e169efc48647e51bb86c2a1ec718R596)

**Comment by miklezzzz** [comment]
@libmonsoon-dev there are some conflicts in the code

**Review by yalosev — APPROVED** [review]

**Pull request merged**